src/chapters/comm_brevity.tex

```diff
@@ -0,0 +1,260 @@
+    Radar locked to a known FRIENDLY aircraft. In response to BUDDY SPIKE.
+    
+    \blueitem[BUDDY SPIKE] 
+    FRIENDLY system radar lock-on indication on radar warning receiver. 
+    Add own ship information to the end.
+    
+    \medskip
+    \textbf{Example}
+    \begin{center}
+        \begin{minipage}{0.9\textwidth}
+            SIMBA 1, BUDDY SPIKE, F-15,
+            BULLSEYE 240/20,
+            TWENTY-THOUSAND,
+            TRACK NORTH.\newline
+            NALA 1, BUDDY LOCK.
+        \end{minipage}
+    \end{center}
+
+    \blueitem[BULLSEYE] 
+    A pre-established reference point on the theater map, from which the position 
+    of an object can be referenced using bearin and range.
+
+    \blueitem[BUSTER] 
+    Use max continous (mil) power.
+
+    \medskip
+    \textbf{Example}
+    \begin{center}
+        \begin{minipage}{0.9\textwidth}
+            SIMBA 1, MUFASA, FLY HEADING 270, BUSTER.
+        \end{minipage}
+    \end{center}
+
+    \blueitem[CHEAPSHOT] 
+    FOX 3 missile datalink terminated between high and medium MPRF.
+
+    \blueitem[COMMIT] 
+    Intercept the GROUPS of interest.
+
+    \medskip
+    \textbf{Example}
+    \begin{center}
+        \begin{minipage}{0.9\textwidth}
+            SIMBA 1, COMMIT NORTH GROUP.
+        \end{minipage}
+    \end{center}
+
+    \blueitem[DECLARE] 
+    Request to ID specified GROUP. Responses include: 
+    FRIENDLY,BOGEY, BANDIT, HOSTILE, NEUTRAL,
+    UNABLE, CLEAN, or FURBALL. 
+
+    \medskip
+    \textbf{Example}
+    \begin{center}
+        \begin{minipage}{0.9\textwidth}
+            MUFASA, SIMBA 1, DECLARE CONTACT BULLSEYE 240/20.
+            SIMBA 1, CONTACT HOSTILE, TYPE FULCRUM, MANUEVER HOT.
+        \end{minipage}
+    \end{center}
+
+    \blueitem[DEFENSIVE] 
+    Aircraft is manuevering defensively, unable to ensure mutual support. 
+
+    \medskip
+    \textbf{Example}
+    \begin{center}
+        \begin{minipage}{0.9\textwidth}
+            SIMBA 1, DEFENSIVE SA-6!
+        \end{minipage}
+    \end{center}
+
+    \blueitem[FADED] 
+    Sensor data is lost on GROUP or CONTACT.
+
+    \blueitem[FOX 2]
+    Indicates launch of IR missile,
+    followed by target information.
+
+    \medskip
+    \textbf{Example}
+    \begin{center}
+        \begin{minipage}{0.9\textwidth}
+            SIMBA 1, FOX 2 HIGH, TURNING LEFT, BANDIT.
+        \end{minipage}
+    \end{center}
+
+    \blueitem[FOX 3]
+    Indicates launch of active-radar missile,
+    followed by target information.
+
+    \medskip
+    \textbf{Example}
+    \begin{center}
+        \begin{minipage}{0.9\textwidth}
+            SIMBA 1, FOX 3 LEAD GROUP BULLSEYE 175/37, 
+            TWENTY-FOUR THOUSAND,
+            TRACK NORTH, 
+            HOSTILE.
+        \end{minipage}
+    \end{center}
+
+    \blueitem[FOX 3]
+    Indicates launch of active-radar missile,
+    followed by target information.
+
+    \medskip
+    \textbf{Example}
+    \begin{center}
+        \begin{tabular}{>{\itshape}r l}
```

Comment:
I would like to see how some other terms look like with this table-based formatting, sometimes it is unclear in the others who is saying what to whom with what modifiers